.github/workflows/release-publish.yaml

```
@@ -0,0 +1,66 @@
+name: Release
+on:
+  push:
+    tags:
+      # These tags should be protected, remember to enable the rule:
+      # https://github.com/canonical/craft-application/settings/tag_protection
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+permissions:
+  contents: write
+
+jobs:
+  source-wheel:
+    runs-on: [self-hosted, jammy]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Fetch tag annotations
+        run: |
+          git fetch --force --tags --depth 1
```

Is there a reason to fetch it using CLI  instead of the action itself ?
```suggestion
        uses: actions/checkout@v4
        fetch-tags: true
      - name: Fetch tag annotations
        run: |
```

---

good point! I'll update it